tests/ssl-opt.sh

```diff
@@ -10391,7 +10391,7 @@ run_test    "TLS 1.3: Client authentication, client alg not in server list - ope
             -c "client state: MBEDTLS_SSL_CLIENT_CERTIFICATE" \
             -c "client state: MBEDTLS_SSL_CLIENT_CERTIFICATE_VERIFY" \
             -c "signature algorithm not in received or offered list." \
-            -C "unkown pk type"
+            -C "unknown pk type"
```

Comment:
this isn't a typo in a comment - is this value used anywhere that fixing this could cause a problem? (or is it related to the 'UNKOWN' above?)

Reply:
This references an error message that was present in ssl_tls13_generic.c but was replaced with "unknown signature type". I suspect this test should be updated to match - @yuhaoth do you agree?